Luma Bridge—Updated for ACO customers only (#493)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/discovery/luma-bridge.mdx

import Badge from '@components/overrides/Badge.astro';
import Embed from '@components/Embed.astro';
import { Icon, Steps } from '@astrojs/starlight/components';
import OptionsTable from '@components/OptionsTable.astro';

This section provides guidance on how to implement the Luma Bridge for your Adobe Commerce on Edge Delivery Services project. 

<Badge text="PaaS" variant="note" />

:::caution

> **Supported Configuration**: Luma Bridge is intended for PaaS instances with complex cart, checkout, and account page implementations that need to be migrated progressively to Edge Delivery Services. It is only supported for the following configuration: **Edge Delivery Services (EDS) + Adobe Commerce Optimizer (ACO) + PaaS**.

Reviewer: You haven't defined PaaS on this page.

:::

## Demo site

The following demo site was built with Luma Bridge. You can access the full demo site at https://mcprod.eds.ecg.magento.com/.

<Embed src="https://mcprod.eds.ecg.magento.com/" height="520" />

## Requirements for implementation

You must meet the following requirements to implement Luma Bridge:

- Both the Edge Delivery Services storefront and the theme-based Adobe Commerce storefront must operate on the same top-level domain.
- You must use the user authentication and cart drop-in components to manage your storefront authentication and cart context. This ensures that Luma Bridge works with the correct set of cookies.

## How does session management work?

Luma Bridge relies on cookies generated by the [user authentication](/dropins/user-auth/) and [cart](/dropins/cart/) drop-in components. These components are part of your Commerce boilerplate template-based storefront on Edge Delivery Services.

> Luma Bridge is a PHP module installed in Adobe Commerce, as noted above. When you enable Luma Bridge in the Adobe Commerce Admin configuration, Adobe Commerce can recognize and create the same cookies that your storefront drop-in components generate. This enables both applications to share the same session cookies, whether the user has a guest session or a logged-in session.

Reviewer: You enable the module with the `bin/magento module:enable Magento_LumaBridge` command mentioned at the bottom of this topic. If you have to enable functionality from the Admin as well, you need to state the configuration path.

```suggestion
Luma Bridge is a PHP module installed in Adobe Commerce, as noted above. When you enable Luma Bridge in the Adobe Commerce Admin configuration, Adobe Commerce can recognize and create the same cookies that your storefront drop-in components generate. This enables both applications to share the same session cookies for both guests and logged-in customers.
```

The cookies include:

- A JWT token that represents the customer session when the user is logged in, generated as a response to the `generateCustomerToken` GraphQL mutation.
- A cart ID that represents a masked quote ID, generated as a response to the `createGuestCart` GraphQL mutation.

No changes are required on your Edge Delivery Services storefront when you use the user authentication and cart drop-in components.

## Implementation considerations

> Before starting an implementation, clarify the following information:

Reviewer: You mention cart, checkout, and account pages in all 3 bulleted items and the trailing paragraph. Tighten this up.
